fxa-auth-db-server/lib/bufferize.js

```diff
@@ -15,21 +13,24 @@ function unbuffer(object) {
   return object
 }
 
-function bufferize(object, ignore) {
+function bufferize(object, onlyTheseKeys) {
   var keys = Object.keys(object)
+  if (onlyTheseKeys) {
+    keys = keys.filter(function(key) {
+      return key in onlyTheseKeys
+    })
+  }
   for (var i = 0; i < keys.length; i++) {
     var key = keys[i]
     var value = object[key]
-    if (ignore.indexOf(key) === -1 && typeof value === 'string' && HEX_STRING.test(value)) {
-      object[key] = Buffer(value, 'hex')
-    }
+    object[key] = Buffer(value, 'hex')
```

Comment:
Side note: using these factory methods is the new new thing - https://nodejs.org/dist/latest-v6.x/docs/api/buffer.html#buffer_buffer_from_buffer_alloc_and_buffer_allocunsafe